src/server/game/Handlers/ArenaTeamHandler.cpp

```diff
@@ -145,6 +146,11 @@ void WorldSession::HandleArenaTeamInviteOpcode(WorldPacket & recvData)
         return;
     }
 
+    //check for fake packets and disaster addons what make client crash
```

Comment:
please reword in `// check for fake packets and bad addons that cause client to crash`